src/mgmt/srv_target.c

```diff
@@ -125,6 +126,7 @@ dir_fsync(const char *path)
 	}
 
 	(void)close(fd);
+	sched_yield();
```

Comment:
The dir_sync() could be called from an ULT on xstream 0 (see ds_mgmt_hdlr_tgt_destroy()), do you intent do make xstream 0 giving up CPU?

Reply:
Good point. I mostly try to release destroy path also.